src/test/resources/pacts/publicapi-direct-debit-connector-search-mandates.json

```diff
@@ -0,0 +1,195 @@
+{
+  "provider": {
+    "name": "direct-debit-connector"
+  },
+  "consumer": {
+    "name": "publicapi"
+  },
+  "interactions": [
+    {
+      "description": "search for mandates",
+      "providerStates": [
+        {
+          "name": "a gateway account with external id and a confirmed mandate exists",
+          "params": {
+            "gateway_account_id": "9ddfcc27-acf5-43f9-92d5-52247540714c",
+            "mandate_id": "test_mandate_id_xyz",
+            "bank_mandate_reference": "410104",
+            "unique_identifier": "MD1234"
+          }
+        }
+      ],
+      "request": {
+        "method": "GET",
+        "path": "/v1/api/accounts/9ddfcc27-acf5-43f9-92d5-52247540714c/mandates",
+        "query": "page=1&display_size=500&bank_statement_reference=410104"
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/json"
+        },
+        "body": {
+          "total" : 1,
+          "count" : 1,
+          "page": 1,
+          "_links": {
+            "self": {
+              "href": "http://localhost:1234/v1/api/accounts/2po9ycynwq8yxdgg2qwq9e9qpyrtre/mandate?page=1&display_size=500&bank_statement_reference=expectedBankStatementReference"
+            },
+            "last_page": {
+              "href": "http://localhost:1234/v1/api/accounts/2po9ycynwq8yxdgg2qwq9e9qpyrtre/mandate?page=1&display_size=500&bank_statement_reference=expectedBankStatementReference"
+            },
+            "first_page": {
+              "href": "http://localhost:1234/v1/api/accounts/2po9ycynwq8yxdgg2qwq9e9qpyrtre/mandate?page=1&display_size=500&bank_statement_reference=expectedBankStatementReference"
+            }
+          },
+          "results": [
+            {
+              "mandate_id": "test_mandate_id_xyz",
+              "mandate_reference": "410104",
+              "provider_id": "MD1234",
+              "service_reference": "test_service_reference",
+              "return_url": "https://example.com/return",
+              "created_date": "2016-01-01T12:00:00Z",
+              "state": {
+                "status": "created",
+                "finished": false,
+                "details": "example details"
```

Comment:
Could you leave this out for now please? I’m making it show real information as part of PP-5442 and it’s easier if the Pact doesn’t yet expect anything.

Reply:
done.